templates/careers/product.html

```diff
@@ -0,0 +1,95 @@
+      </div>
+      <div class="col-6">
+        <p>As product manager,  you are a representative of Canonical and you are an advocate of our brands and products. We all work together across the full portfolio - you are expected to understand the big picture of how Canonical's products fit together, there are no siloes. These are not behind-the-scenes roles, they are public leadership roles. We expect our team to be just as comfortable delivering our message online as they are face to face at meetups, meetings, conferences, partner workshops or press events. This team is all about the excellence, quality and thought-leadership required to put us squarely into the frame for companies thinking afresh about their biggest technology questions.</p>
+      </div>
+    </div>
+    
+  </div>
+  <div class="p-strip is-shallow u-no-padding--top">
+    <hr class="is-fixed-width u-no-margin--bottom">
+    <div class="row">
+      <div class="col-6">
+        <h3 class="p-heading--2">Who you are</h3>
+      </div>
+      <div class="col-6">
+        <p>You are customer obsessed. You over-communicate strategy and vision.  You are strategic and tactical. You use data and your intuition to make decisions.  You love sharing ideas, information and enthusiasm. You partner with and influence the entire organisation &ndash; and are passionate about the future of technology.</p>
+        <p>Our team has an insatiable intellectual curiosity. We love the question “Why?” and are relentless in finding answers through user interviews, data, and experimentation.</p>
+        <p>In this team, it's important to have a sense of ownership. Every part of the product's success is your responsibility. You will work closely with the CEO and collaborate with every part of the organisation to realise that success &ndash; whether it's marketing, sales, engineering, or any other key stakeholder.</p>
+        <p>We look for people who are excellent communicators and can speak to and write effectively for everyone - from marketers to engineers to c-suite executives. You communicate in the format the audience best receives information, whether that's written, visually, or with data. Through it all, you communicate with confidence and empathy.</p>
+        <p>You take a strategic approach and use the available resources to deliver as much value to the business and customers as possible. You understand the long game and spend a lot of time looking around corners, envisioning what might go right and what might go wrong.</p>
+        <p>Through it all, you are process-driven and focused. You'll keep our product teams on track by documenting dependencies, planning sprints, and staying ahead of issues that might block our progress.</p>
+    </div>
+    </div>
+  </div>  
+</section>
+
+{% include "shared/_social-icons-section.html" %}
+
+{% with section_classes='p-strip', heading_topic='Product', tag_id='1216', limit='3' %}
+  {% include "shared/_blog-card.html" %}
```

Comment:
From an accessibility standpoint, it is really not clear that the headings of these blog  are links. Especially as other links on the page are blue. At minimum we should be underlining the heading, or preferably matching the style of the other links (although this has its own problem for people with color blindness)
![image](https://user-images.githubusercontent.com/58276363/218482482-2a7d93c7-a1cb-47da-8650-56425e9717ee.png)